<launch>
    <arg name="robot_sim" default = "false" />
    <arg name="qb_hand" default = "true" />

    <rosparam command="load" 
    file="$(find impose_grasp)/config/params.yaml" />
    
    <!-- Launch fake robot in the scene -->
    <include unless="$(arg robot_sim)" file="$(find panda_qb_description)/launch/real_panda.launch">
        <arg name="qb_hand" value="$(arg qb_hand)" />
    </include>

    <include if="$(arg robot_sim)" file="$(find panda_qb_description)/launch/sim_panda.launch">
        <arg name="qb_hand" value="$(arg qb_hand)" />
    </include>
</launch>